reana/reana_benchmark/cli.py

```diff
@@ -165,6 +201,14 @@ def launch(
 @reana_benchmark.command(name="analyze")
 @workflow_option
 @workflow_range_option
+@click.option(
+    "--datetime",
```

Comment:
*thought:* I was thinking maybe splitting the command into `--datetime-start`, `--datetime-end` (or similar like mentioned [here](https://github.com/reanahub/reana/issues/583#issuecomment-975318100)). Having a single long string feels a bit intimidating to me. Let's see what review will yield.

In the case of splitting, we might need to consider a scenario where only one limit is given (or enforce both).

Reply:
I personally like that it's defined in one command, it's pretty easy to simply copy the value from the example you provided and edit it as one needs. I think we could also add `-d` for `--datetime`, WDYT?

Reply:
I can add `-d`. It will also follow other options.

Reply:
Regarding splitting the command, let's leave it as it is. In case it will bring pain we can always split the command.